app/helpers/agent_helper.rb

```diff
@@ -169,7 +169,8 @@ def agent_tooltip(agent)
     if agent.affiliations && agent.affiliations != []
       affiliations = ""
       agent.affiliations.each do |affiliation|
-        affiliations = affiliations + affiliation.acronym + " "
```

Comment:
change to 
```
affiliations = "#{affiliations} #{affiliation.acronym} "
```

Reply:
and clicking on save does it work?

Reply:
Yes it works

Reply:
does the bellow version work ?, if yes use it, you will gain one line.
```
affiliations = "#{affiliations} #{ affiliation.acronym || affiliation.name }"
```

Reply:
Yes working, thanks